index.html

```diff
@@ -107,8 +107,7 @@ <h3>Portfolio Template</h3>
             <div class="work__text">
               <h3>Calculator</h3>
               <p>
-                Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod,
-                eius.
+                The most basic calculators, which can cost pennies to purchase, add, subtract, multiply and divide.
```

Comment:
This looks good  What do you say about:

```suggestion
                A basic calculator app which can perform basic arthimetic opreations.
```

Reply:
thanks